tests/compas_timber/test_assembly.py

```diff
@@ -84,52 +62,29 @@ def test_deepcopy(mocker):
     F2 = Frame(Point(0, 0, 0), Vector(1, 0, 0), Vector(0, 1, 0))
     B1 = Beam(F1, length=1.0, width=0.1, height=0.12)
     B2 = Beam(F2, length=1.0, width=0.1, height=0.12)
-    A = TimberAssembly()
+    A = TimberModel()
     A.add_beam(B1)
     A.add_beam(B2)
     _ = LButtJoint.create(A, B1, B2)
 
-    A_copy = deepcopy(A)
+    A_copy = A.copy()
     assert A_copy is not A
     assert A_copy.beams[0] is not A.beams[0]
 
 
-def test_find():
-    A = TimberAssembly()
-    B = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
-    A.add_beam(B)
-    assert B == A.find(B.guid)
-
-
-def test_parts_joined(mocker):
-    mocker.patch("compas_timber.connections.LButtJoint.add_features")
-    A = TimberAssembly()
-    B1 = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
-    B2 = Beam(Frame.worldYZ(), length=1.0, width=0.1, height=0.1)
-    B3 = Beam(Frame.worldZX(), length=1.0, width=0.1, height=0.1)
-
-    A.add_beam(B1)
-    A.add_beam(B2)
-    A.add_beam(B3)
-    _ = LButtJoint.create(A, B1, B2)
-
-    assert A.are_parts_joined([B1, B2])
-    assert not A.are_parts_joined([B1, B3])
-
-
 def test_beams_have_keys_after_serialization():
-    A = TimberAssembly()
+    A = TimberModel()
     B1 = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
     B2 = Beam(Frame.worldYZ(), length=1.0, width=0.1, height=0.1)
     B3 = Beam(Frame.worldZX(), length=1.0, width=0.1, height=0.1)
     A.add_beam(B1)
     A.add_beam(B2)
     A.add_beam(B3)
-    keys = [beam.key for beam in A.beams]
+    keys = [beam.guid for beam in A.beams]
 
     A = json_loads(json_dumps(A))
 
-    assert keys == [beam.key for beam in A.beams]
+    assert keys == [beam.guid for beam in A.beams]
```

Comment:
I agree. I realized there's a `graph_node` attribute kinda later on. will change everywhere to use that.

Reply:
Ok so did another pass. within CT, only `guid` should be used. `graph_node` is sometimes used in tests where it would otherwise be a pain in the ass to use guids (e.g. the neighboring beam test).